pkg/provider/vsphere/vsphere.go

```diff
@@ -151,15 +149,15 @@ func parseParameters(properties map[string]interface{}, p *plugin) (vmInstance,
 		return newInstance, errors.New("Property 'Datastore' must be set")
 	}
 	*p.vC.dsName = properties["Datastore"].(string)
-	log.Debugf("Datastore set to %s", *p.vC.dsName)
+	log.Debug(fmt.Sprintf("Datastore set to %s", *p.vC.dsName))
```

Comment:
The convention for log15 (the library we use) is 
```
log.Debug("a general message", "key1", value1, "key2", value2)
```
So this log statement should be rewritten to
```
log.Debug("Setting datastore", "dataStore", *p.vC.dsName)
```